docs/overview/deep_dive/inplace_updates.rst

```diff
@@ -399,20 +407,23 @@ The second and third points are the most important points.
 
 We'll use :func:`ivy.cross_entropy` as an example:
 
+
 .. code-block:: python
 
     def cross_entropy(
-        true: Union[ivy.Array, ivy.NativeArray],
-        pred: Union[ivy.Array, ivy.NativeArray],
-        /,
-        *,
-        axis: Optional[int] = -1,
-        epsilon: float =1e-7,
-        out: Optional[ivy.Array] = None
+    true: Union[ivy.Array, ivy.NativeArray],
+    pred: Union[ivy.Array, ivy.NativeArray],
+    /,
+    *,
+    axis: int = -1,
+    epsilon: float = 1e-7,
+    reduction: str = "sum",
+    out: Optional[ivy.Array] = None,
     ) -> ivy.Array:
+        ivy.utils.assertions.check_elem_in_list(reduction, ["none", "sum", "mean"])
         pred = ivy.clip(pred, epsilon, 1 - epsilon)
         log_pred = ivy.log(pred)
-        return ivy.negative(ivy.sum(log_pred * true, axis, out=out), out=out)
+        return _reduce_loss(reduction, log_pred * true, axis, out)
```

Comment:
let's pass `out` as a keyword argument following convention

Reply:
I didn't get what you want me to do in this section ???

Reply:
Also I think here as the code changed we need to edit the text right ?
```suggestion
Firstly, the return of :func:`ivy.sum` is the same shape and type as the return of the entire function, and so we can also write this output to the :code:`out` argument inplace.
```